tensorrt_llm/llmapi/llm_args.py

```diff
@@ -986,6 +1001,31 @@ def _check_executor_config_options_consistency():
             llm_args_attr
         ), f"New options found in underlying ExecutorConfig: {llm_args_attr - executor_config_attrs}"
 
+        # ensure build_config and LlmArgs consistency
+        if kwargs_dict.get("backend") == "pytorch":
+            if "build_config" in kwargs_dict:
+                print_colored((
```

Comment:
I think in PyTorch backend, it still needs `BuildConfig.PluginConfig.tokens_per_block` to set the page size of KV cache now. So, I suggest we remove this warning in this MR. After we clean up all configs, we can add an assertion directly.

```
assert "build_config" not i kwargs_dict, "xxx"
```

Reply:
The `tokens_per_node` hasn't been moved to the Python `KvCacheConfig`, right? @hchings 
After it is moved there, I think we can add an `assert` as Jun mentioned.

Reply:
Briefly discussed w/ @Superjomn offline, will hold off moving `tokens_per_block` to `KVCacheConfig` untill `KVCacheConfig` no longer depends on pybind. 

This will be after Torch path successfully switch to LlmArgs from ExecutorConfig.